Sources/GDSAnalytics/Events/Forms/FormEvent.swift

```diff
@@ -13,9 +13,14 @@ public struct FormEvent: Event {
         ].mapValues(\.formattedAsParameter)
     }
     
-    public init(textKey: String, _ variableKeys: String...,
+    public init(name: EventName = .formResponse,
```

Comment:
as above, I think making this flexible undermines the point of them being distinct types.